systemtests/src/main/java/io/enmasse/systemtest/amqp/AmqpClient.java

```diff
@@ -200,16 +200,14 @@ public void close() throws Exception {
         CompletableFuture<Void> connectPromise = new CompletableFuture<>();
         vertx.deployVerticle(new Sender(options, new LinkOptions(options.getTerminusFactory().getSource(address), options.getTerminusFactory().getTarget(address), Optional.empty()), messages, predicate, connectPromise, resultPromise, containerId));
 
-        try {
-            connectPromise.get(2, TimeUnit.MINUTES);
+        try { connectPromise.get(2, TimeUnit.MINUTES);
```

Comment:
unnecessary reformat
```suggestion
        try {
                connectPromise.get(2, TimeUnit.MINUTES);
```